independent-projects/tools/devtools-common/src/main/java/io/quarkus/registry/RepositoryIndexer.java

```diff
@@ -38,6 +38,7 @@ public void index(Repository repository, IndexVisitor visitor) throws IOExceptio
                 io.quarkus.dependencies.Extension ext = artifactResolver.resolveExtension(extension, release);
                 if (ext != null) {
                     visitor.visitExtension(ext, release.getQuarkusCore());
+                    visitor.visitExtensionRelease(ext.getGroupId(), ext.getArtifactId(), release);
```

Comment:
I think this would do the same thing?

 ```suggestion
 					for (String version: release.getCompatibleQuarkusCore()) {
                    		visitor.visitExtension(ext, version);
                   	}
```